bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -289,6 +311,15 @@ channel-type.config.netatmo.live-stream-url.quality.option.low = Low Quality
 channel-type.config.netatmo.live-stream-url.quality.option.poor = Poor Quality
 channel-type.config.netatmo.live-stream-url.quality.option.high = High Quality
 
+# channel group types
+
+channel-group-type.netatmo.person-event.label = Last Event
```

Comment:
All these keys are outdated? 
To be removed if confirmed.

Reply:
Yes they now are since I unified last-event with person-event.